src/model/WorkflowState.model.js

```diff
@@ -51,4 +50,16 @@ class WorkflowState extends Enumify {
   }
 }
 
+/**
+ * Workflow states ordered for display purposes.
+ */
+export const WorkflowStateOrder = [
+  WorkflowState.RUNNING,
+  WorkflowState.PAUSED,
+  WorkflowState.STOPPING,
+  WorkflowState.STOPPED,
+  WorkflowState.INSTALLED,
+  WorkflowState.ERROR
+]
```

Comment:
Matching what we have in `TaskState.model.js`, where the enum order is managed in an array, instead of the order of elements declared in the enum.